modules/wpml/wpml-config.php

```diff
@@ -849,4 +826,136 @@ private function normalize_path( string $path ): string {
 
 		return $path;
 	}
+
+	/**
+	 * Adds (or removes) meta names to the list of metas to copy or synchronize.
+	 *
+	 * @since 3.6
+	 *
+	 * @param string[] $metas The list of meta names to copy or synchronize.
+	 * @param string   $xpath Xpath to the meta fields in the xml files.
+	 * @param bool     $sync  Either sync is enabled or not.
+	 * @return string[]
+	 *
+	 * @phpstan-param array<non-falsy-string> $metas
+	 * @phpstan-param non-falsy-string $xpath
+	 */
+	private function filter_metas_to_copy( array $metas, string $xpath, bool $sync ): array {
+		$parsed_metas    = $this->parse_xml_metas( $xpath );
+		$metas_to_remove = array();
+
+		foreach ( $parsed_metas as $parsed_meta ) {
+			if ( 'copy' === $parsed_meta['action'] || ( ! $sync && in_array( $parsed_meta['action'], array( 'translate', 'copy-once' ), true ) ) ) {
```

Comment:
```suggestion
                         $do_sync =  'copy' === $parsed_meta['action'];
                         $do_copy = ! $sync && in_array( $parsed_meta['action'], array( 'translate', 'copy-once' ), true ) );
			if ( $do_sync || $do_copy ) {
```
Do you like it?

Reply:
More readable indeed but we slightly lose in perfs.
- Current case: if action is `'copy'`, we go directly into the `if`.
- Suggested case: if action is `'copy'`, we still need to evaluate `$do_copy`.

So I don't know ^^'

Reply:
I'll let @Chouby make a decision here 😅